cognee/modules/graph/cognee_graph/CogneeGraph.py

```
@@ -0,0 +1,119 @@
+                    raise ValueError(f"Edge references nonexistent nodes: {source_id} -> {target_id}")
+
+        except (ValueError, TypeError) as e:
+            print(f"Error projecting graph: {e}")
+        except Exception as ex:
+            print(f"Unexpected error: {ex}")
+
+
+"""
+The following code only used for test purposes and will be deleted later
+"""
+import asyncio
```

Ruff

Module level import not at top of file

(E402)